openapi/dcos.yaml

```diff
@@ -2163,10 +2163,9 @@ components:
       properties:
         command:
           $ref: '#/components/schemas/command'
-        # Causes a JSON unmarshal error at the moment, config should be defined as a JSON schema instead.
-        # config:
-        #   additionalProperties: true
-        #   type: object
+        config:
+          additionalProperties: false
```

Comment:
You can actually remove it completely as it is with other properties.

Reply:
I've added a comment as it should be defined as a JSON schema eventually.